sdk/identity/identity/src/msal/nodeFlows/msalNodeCommon.ts

```diff
@@ -255,6 +299,18 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
       };
     }
 
+    if (hasNativeBroker() && this.enableBroker) {
+      this.msalConfig.broker = {
+        nativeBrokerPlugin: nativeBrokerInfo!.broker,
+      };
+      if (!this.parentWindowHandle) {
+        // error should have been thrown from within the constructor of Interactive Browser Credential
+        this.logger.warning(
+          "Parent window handle is not specified for the broker. This may cause unexpected behavior.Please provide the parentWindowHandle"
```

Comment:
```suggestion
          "Parent window handle is not specified for the broker. This may cause unexpected behavior. Please provide the parentWindowHandle."
```